things/redis/streams_test.go

```diff
@@ -186,8 +186,8 @@ func TestListThings(t *testing.T) {
 	require.Nil(t, err, fmt.Sprintf("unexpected error %s", err))
 
 	essvc := redis.NewEventStoreMiddleware(svc, redisClient)
-	esths, eserr := essvc.ListThings(token, 0, 10)
-	ths, err := svc.ListThings(token, 0, 10)
+	esths, eserr := essvc.ListThings(token, 0, 10, "")
+	ths, err := svc.ListThings(token, 0, 10, "lora")
```

Comment:
Can use change this to something else. It's not a huge deal, but I don't like having anything protocol specific even in tests.

Reply:
Good catch, I forgot this one